charts/hivemq-platform/tests/configuration/hivemq_rest_api_listeners_configuration_test.yaml

```diff
@@ -0,0 +1,105 @@
+suite: HiveMQ Platform Configuration - REST API listeners tests
```

Comment:
Should we have a failing test when someone tries to use `hivemqListenerName` with `type: rest-api`?

Reply:
Nice catch!